### PR TITLE
Smoke test dark theme

### PR DESCRIPTION
**Pull request by lapp0**
fixes https://github.com/nix-gui/nix-gui/issues/175

provided by @milahu https://github.com/nix-gui/nix-gui/pull/187#issuecomment-974237002

### CODE REVIEW

File: nixui/options/option_tree.py [hunk 1]
```diff
@@ -256,6 +272,13 @@ def children(self, attribute, mode="direct"):
             if '"<name>"' not in node.tag
         }
 
+    @functools.lru_cache(100000)  # this breaks when nixos has 100,000 attributes
```
Comment (milahu) [inline nixui/options/option_tree.py:275]: > this breaks when nixos has 100,000 attributes

so its not a "cache" but a store?

File: nixui/graphics/navlist.py [hunk 2]
```diff
@@ -20,6 +20,130 @@ def __new__(cls, statemodel, set_option_path_fn, option_path, option_type=None,
...
+        text = data['text'] + (f" ({data['child_count']})" if data.get('child_count') else "")
+
+        # draw subtext
+        if data.get('extra_text'):
+            painter.save()
+            extra_text_rect = QtCore.QRect(text_rect)
+            extra_text_rect.setY(text_rect.y() + option.rect.height() / 3)
+            text_rect.setY(text_rect.y() - option.rect.height() / 3)  # adjust the base text upwards
+            extra_text_font = QtGui.QFont()
+            extra_text_font.setItalic(True)
```
Comment (milahu) [inline nixui/graphics/navlist.py:75]: hard to read

Comment (lapp0) [reply nixui/graphics/navlist.py:75]: This PR bundles a lot of changes stacked together. This is part of https://github.com/nix-gui/nix-gui/pull/187/files

I'll change it there.

File: nixui/options/option_tree.py (hunk 1 above)
Comment (lapp0) [reply nixui/options/option_tree.py:275]: Yes

Comment (lapp0) [reply nixui/options/option_tree.py]: I've merged a few PRs recently, if you'd like to review them you're more than welcome to so I can make any necessary changes. They're all relatively low risk though.

**Pull request merged**